src/main/java/reposense/util/FileUtil.java

```diff
@@ -168,6 +171,20 @@ public static void createDirectory(Path dest) throws IOException {
         Files.createDirectories(dest);
     }
 
+    /**
+     * Returns true if {@code path} is a valid path.
+     * Produces log messages when the invalid file path is skipped.
+     */
+    public static boolean isValidPath(String path) {
+        try {
+            Paths.get(path);
+        } catch (InvalidPathException | NullPointerException ex) {
```

Comment:
When will `NullPointerException` be thrown? `path` is null?

Reply:
My bad, `NullPointerException` won't be thrown under any circumstances, and we won't need to deal with a null input `path`.